elasticai/creator/ir/node.py

```diff
@@ -0,0 +1,23 @@
+from typing import overload
+
+from .abstract_ir_data import AbstractIRData
+from .abstract_ir_data.mandatory_field import MandatoryField
+from .attribute import AttributeT
+
+
+class Node(AbstractIRData):
```

Comment:
Maybe split mandatory fields class in transformable mandatory fields and mandatory fields